Doctrine/Orm/DataProvider.php

```diff
@@ -67,7 +67,7 @@ public function getItem($id, $fetchData = false)
     /**
      * {@inheritdoc}
      */
-    public function getCollection($page, array $filters, $itemsPerPage = 30, $order = null)
+    public function getCollection($page, array $filters, $itemsPerPage = 30, array $order = null)
```

Comment:
`$order` will never be null. To disable order we just need to pass an empty array. The must can be changed to `array $order = []`